js/index.js

```diff
@@ -1,7 +1,39 @@
 // Iteration 1: Names and Input
+let hacker1 = "Mayhem";
+console.log(`The driver's name is ${hacker1}`)
 
+let hacker2 = "Chai Latte"
+console.log (`The navigator's name is ${hacker2}`)
 
 // Iteration 2: Conditionals
 
+if (hacker1.length > hacker2.length) {
+    console.log(`The driver has the longest name, it has ${hacker1.lenght} characters`);} 
```

Comment:
Update this line to "length" instead of "lenght". Otherwise, the return value here will be `undefined` instead of the actual value

```suggestion
    console.log(`The driver has the longest name, it has ${hacker1.length} characters`);} 
```